### DISCUSSION TITLE
Updates on Org Roam compatibility

### DISCUSSION THREAD

**Discussion by JLucasGaldino** [amake/orgro#101] [General]
Hello. Are there any updates on making Orgro work for following id-based links across files, like on Org roam? Thanks for the amazing app!

**Comment by amake** [comment]
No, I haven't done any further work on it.

**Comment by amake** [comment]
OK I lied; I looked back and found a stash from 2021 with a partial implementation of ID search for iOS.

Contrary to https://github.com/amake/orgro/discussions/44#discussioncomment-741144 I decided to do the dumbest thing that works: full search of all files, every time, with no caching.

The result will be available for testing in v1.40.0 soon:

- [iOS](https://testflight.apple.com/join/8YTxcuZC)
- [Android](https://play.google.com/store/apps/details?id=com.madlonkay.orgro)

**Reply by amake** [reply]
v1.40.4 is now available on all platforms.

**Comment by amake** [comment]
This discussion thread seems redundant with #17. If there's nothing else to discuss here then I will close.
